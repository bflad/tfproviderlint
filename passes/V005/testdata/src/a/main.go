package a

import (
	"github.com/hashicorp/terraform-plugin-sdk/helper/validation"
)

func f() {
	// Passing

	_ = validation.StringIsJSON

	// Comment ignored

	//lintignore:V005
	_ = validation.ValidateJsonString

	_ = validation.ValidateJsonString //lintignore:V005

	// Failing

	_ = validation.ValidateJsonString // want "deprecated validation.ValidateJsonString should be replaced with validation.StringIsJSON"
}
