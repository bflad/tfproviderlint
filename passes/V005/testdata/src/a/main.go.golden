package a

import (
	"github.com/hashicorp/terraform-plugin-sdk/helper/validation"
)

func f() {
	// Passing

	_ = validation.StringIsJSON

	// Comment ignored

	//lintignore:V005
	_ = validation.ValidateJsonString

	_ = validation.ValidateJsonString //lintignore:V005

	// Failing

	_ = validation.StringIsJSON // want "deprecated validation.ValidateJsonString should be replaced with validation.StringIsJSON"
}
